### DISCUSSION TITLE
Toggling class off from v-bind:class

### DISCUSSION THREAD

**Discussion by Abdillah** [vuejs/petite-vue#160] [Ideas]
Currently existing class in a DOM cannot be toggle off / removed using `:class="{ hidden: false }"` as with Vue counterpart.
Can such feature be included in petite-vue?

For example, if we have a dom like these,

```
<div class="hidden" :class="{ hidden: isMenuHidden }">
  <!-- code -->
</div>
```

I would expect the element to have `hidden` class when `isMenuHidden === true` and no `hidden` class (removed) when `isMenuHidden === false`.

**Comment by EmmyMay** [comment]
Hi, Abdillah, can you explain more?

**Reply by Abdillah** [reply]
I've updated the original post for clarity

**Reply by EmmyMay** [reply]
Oh I think I get you now, you mean that there is no dynamic class binding. I am shocked that does not exist tbh. Maybe we can look at the code together and submit a PR?

**Reply by Abdillah** [reply]
Actually no, there *is* dynamic class binding. But, once the class actually existed before, it doesn't toggle..

**Reply by Abdillah** [reply]
I have write these condition in my company fork and willing to upstream it.
But somehow this feature is not align with VueJs (last I checked, VueJs cannot do this too).
